x-pack/plugins/lists/public/exceptions/mock.ts

```diff
@@ -6,27 +6,8 @@
 import {
   CreateExceptionListItemSchemaPartial,
   CreateExceptionListSchemaPartial,
-  ExceptionListItemSchema,
-  ExceptionListSchema,
 } from '../../common/schemas';
 
-export const mockExceptionList: ExceptionListSchema = {
-  _tags: ['endpoint', 'process', 'malware', 'os:linux'],
-  created_at: '2020-04-23T00:19:13.289Z',
-  created_by: 'user_name',
-  description: 'This is a sample endpoint type exception',
-  id: '1',
-  list_id: 'endpoint_list',
-  meta: {},
-  name: 'Sample Endpoint Exception List',
-  namespace_type: 'single',
-  tags: ['user added string for a tag', 'malware'],
-  tie_breaker_id: '77fd1909-6786-428a-a671-30229a719c1f',
-  type: 'endpoint',
-  updated_at: '2020-04-23T00:19:13.289Z',
-  updated_by: 'user_name',
-};
-
 export const mockNewExceptionList: CreateExceptionListSchemaPartial = {
```

Comment:
I'll clean this up to get rid of this file all together on my PR - thanks so much for updating the tests!